api/response/schema.py

```diff
@@ -251,6 +259,44 @@ def mutate(self, info, responses, resolved):
         )
 
 
+class ArchiveResponse(graphene.Mutation):
+    """
+    Returns payload after archiving a resolved response
+    """
+
+    class Arguments:
+        response_id = graphene.Int(required=True)
+
+    room_response = graphene.Field(ResponseDetail)
+
+    @Auth.user_roles('Admin')
```

Comment:
please add the `Super_Admin` user role here as super admin should be ab;e to perform this task

Reply:
Hi @IssaIan I can't seem to run the migration on your description, did you update your PR?

Reply:
Yes I did

Reply:
Made the requested change.